public-user/playbook.yml

```diff
@@ -41,3 +43,8 @@
 
   vars:
     omero_web_public_password: public
+    omero_server_config_set:
+      omero.db.poolsize: 50
+      omero.jvmcfg.percent.blitz: 50
+      omero.jvmcfg.percent.indexer: 20
+      omero.jvmcfg.percent.pixeldata: 20
```

Comment:
Since we don't know the amount of RAM the end user has at this point, we might be better off leaving out the `omero.jvmcfg.percent` config.

With this split, used without a cap via [omero.jvmcfg.system_memory](https://docs.openmicroscopy.org/omero/5.4.1/sysadmins/config.html#omero-jvmcfg-system-memory), a Linux server will only end up with 500MB to play with, on a server with 5GB of ram. 

Chances are people will try this on a VM with ≈2GB. Perhaps OMERO won't use it's allocation straight away, but it's possibly dicey.

@joshmoore - thoughts?

Reply:
We could use a hard-coded value rather than percent with the minimal required values, then it can fail if that's not available.

Reply:
Not totally sure how that would look in configuration.

Reply:
Wouldn't it be ``bin/omero config set omero.jvmcfg.system_memory 24000``

from https://docs.openmicroscopy.org/omero/5.4.1/sysadmins/server-performance.html#configuration-properties
Whilst keeping the percent strategy as described in the doc
?

Reply:
You can check `ansible_memtotal_mb` to see the amount of memory on the system, e.g. try something like `fail: ... <message> ... when: ansible_memtotal_mb < XXX`

Reply:
Guess the question is what minimum `mb` do we want to support? 4000?

Reply:
The higher you set it the less likely you'll be able to test it with molecule on travis

Reply:
Another alternative would be to comment this section out with info about it being important in production.

Reply:
The commenting sounds quite attractive.

Reply:
@pwalczysko ?

Reply:
@joshmoore : sorry, was already almost on my way to holiday. Yes, agree with commenting the settings of the memory etc. out. Should I open a PR against this branch ?

Reply:
Pushed a comment and a change to disable the properties.